src/auth/index.ts

```diff
@@ -53,20 +55,27 @@ async function login(
     return {
       message: 'Incorrect username or password',
       success: false,
-      token: '', // Won't be needed anymore
+      token: '',
     };
   } else {
-    ctx.session.user = user;
+    const accessToken = sign(
+      {
+        username: user.username,
+      },
+      SECRET_ACCESSTOKEN,
+      { expiresIn: '30m' }
+    ); // token expires in 30 mins
```

Comment:
This seems annoyingly low. For the purposes of this project it's probably fine to set this to something pretty long like 48 hours tbh. Will cut down on annoyances during development and during demos